skills/nodej/bulk-update-skills.js

```diff
@@ -0,0 +1,40 @@
+const execSync = require('child_process').execSync;
+
+let removeSkills = ['Juggling','Fire Breathing','Origami'];
+
+if(process.platform === 'linux'){
+    console.log('Working on it...');
+    // Loop through each skill to be deleted
+    removeSkills.forEach(skill => {
+        userData = execSync('gc users list -a --pageSize=100 --expand="skills" | jq -c -r \'[.[] | select(.skills[].name=="'+ skill +'")]\'');
+
+        var obj = JSON.parse(userData);
+
+        // Loop through each user with skill
+        obj.forEach(user => {
+            skillArr = user.skills.find(x => x.name == skill);
+
+            // Remove skill for each user
+            execSync('gc users skills delete '+ user.id +' '+ skillArr.id +'');
```

Comment:
Is there some error handling that could be done with this function?

Reply:
Added check if there is no returned data from the first gc query